public/style.css

```diff
@@ -209,6 +209,10 @@
     text-indent: 15%;
 }
 
+#mainft {
+    margin-top: 50%;
+}
+
```

Comment:
it already works without the margin I think this is unnecessary but we'll see

Reply:
I just added it in case we need to show our modules by separate at any point